### ISSUE TITLE
done everything properly and it gives me this error, oauth has no problem

### ISSUE THREAD

**Issue by cuhsy** [BOT-HAT/Youtube-First-Comment-Bot#24]
Invalid file format. See https://developers.google.com/api-client-library/python/guide/aaa_client_secrets Expected a JSON object with a single property for a "web" or "installed" application

WARNING: Please configure OAuth 2.0
To make this sample run you will need to populate the client_secrets.json file
found at:
   ...\client_secrets.json
with information from the APIs Console
https://console.developers.google.com
For more information about the client_secrets.json file format, please visit:
https://developers.google.com/api-client-library/python/guide/aaa_client_secrets

**Comment by aousmanebarry** [comment]
lol i feel like i know exactly why you're here but i won't say anything if you happen to not know about it

**Comment by Macon122** [comment]
> lol i feel like i know exactly why you're here but i won't say anything if you happen to not know about it

Is it MrBeast promo

**Comment by Macon122** [comment]
> lol i feel like i know exactly why you're here but i won't say anything if you happen to not know about it

Bro pls help me out

**Comment by aousmanebarry** [comment]
> > lol i feel like i know exactly why you're here but i won't say anything if you happen to not know about it
> 
> Is it MrBeast promo

Yes. I would have helped you out in normal times but I'm trying to win too, sorry.

**Comment by cuhsy** [comment]
> > > lol i feel like i know exactly why you're here but i won't say anything if you happen to not know about it
> > 
> > 
> > Is it MrBeast promo
> 
> Yes. I would have helped you out in normal times but I'm trying to win too, sorry.

cmon bro, there are hundreds of people doing this, and you tryna ruin my shot, cmon, sharing is caring

**Comment by nasko94** [comment]
You need to create a google console account

The interesting error I have is The client secrets were invalid: 
('Error opening file', 'Youtube-First-Comment-Bot-master\\clientsecrets.json', 'No such file or directory', 2)

**Comment by SFM61319** [comment]
> You need to create a google console account
> 
> The interesting error I have is The client secrets were invalid: 
> ('Error opening file', 'Youtube-First-Comment-Bot-master\\clientsecrets.json', 'No such file or directory', 2)

I believe it's `client_secrets.json`. Also, make sure you update the file with the actual JSON data from your Google console account. And if you get a decoding error, pass `encoding='utf8'` to the `open()` file handler (I'm pretty sure the JSON file is UTF-8 encoded).

**Comment by cuhsy** [comment]
> You need to create a google console account
> 
> The interesting error I have is The client secrets were invalid: ('Error opening file', 'Youtube-First-Comment-Bot-master\clientsecrets.json', 'No such file or directory', 2)

Yeah i did that and put the json contents in client_secrets.json and i get the error as i said in the original post

**Comment by lauaall** [comment]
This bot doesn't work anymore.

**Comment by nasko94** [comment]
> I believe it's client_secrets.json. Also, make sure you update the file with the actual JSON data from your Google console account. And if you get a decoding error, pass encoding='utf8' to the open() file handler (I'm pretty sure the JSON file is UTF-8 encoded).

How do you do that?

**Comment by nasko94** [comment]
Okay, I've done it. But the bot should have look for the entire channel not just a playlist. How do I make it a channel id?

**Comment by cuhsy** [comment]
Apparently this does not work anymore. Sad
